dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/NameAliases.java

```diff
@@ -23,11 +23,13 @@
     public NameAliases(String[] originalNames) {
         this.originalNames = originalNames;
         this.aliases = originalNames.clone();
-        this.originalExtensions = new ArrayList<>();
-        this.aliasesExtensions = new ArrayList<>();
+        synchronized (this) {
+            this.originalExtensions = new ArrayList<>();
+            this.aliasesExtensions = new ArrayList<>();
+        }
     }
 
-    public void newAliases(Map<String, String> newAliases) {
+    public synchronized void newAliases(Map<String, String> newAliases) {
```

Comment:
This won't fix the issue... LogTraceData has unsynchronized access to originalExtensions / aliasesExtensions.